cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/containers/NotifyContextRequest.java

```diff
@@ -352,7 +352,16 @@ public String getContextValue() {
                 return  "" + value;
             }  // if then else if
         } // getContextValue
-        
+
+        /**
+         * Gets value.
+         *
+         * @return the value on JsonElement Object.
+         */
+        public JsonElement getValue() {
+            return value;
+        }
```

Comment:
```suggestion
        } // getValue
```

(Looking to other cases, it seems this is done in every function...)